neuro_extras/main.py

```diff
@@ -170,14 +179,17 @@ def image() -> None:
 
 
 @image.command("build")
+@click.option("--build-arg", multiple=True)
 @click.option("-f", "--file", default="Dockerfile")
 @click.argument("path")
 @click.argument("image_uri")
-def image_build(file: str, path: str, image_uri: str) -> None:
-    run_async(_build_image(file, path, image_uri))
+def image_build(file: str, path: str, image_uri: str, build_arg: Sequence[str]) -> None:
```

Comment:
I think order matters here (might be wrong though).

Reply:
It doesn't, but I rearranged options and arguments for consistency.